### PR TITLE
hoc2015: possible fix for ios 8 level 1 showing black squares.

### PR DESCRIPTION
**Pull request by breville**
We've seen on some iOS 8 devices that level 1 shows black squares where obstacles should be shown.  Root cause is not understood, but this is a potential workaround by using a single background image for that level.  The only functional downside is that obstacle z-sorting isn't possible, so they are always drawn behind the main avatar; in reality this probably won't be noticed.

For good measure, this first level also preloads the tile image used in subsequent levels.  It's not yet known whether this is necessary or not.

### CODE REVIEW

**Comment by breville** [comment]
fyi @cpirich @bcjordan @philbogle @alicesteinglass

**Comment by islemaster** [comment]
Looks good at a glance.  Gimme a minute, will pull and try it out.

**Comment by islemaster** [comment]
Tested locally, LGTM.  The layering stuff isn't even that noticeable when you do go off the beaten path, because we still have collision.

File: apps/src/studio/studio.js [hunk 1]
```diff
@@ -3244,6 +3253,12 @@ Studio.drawWallTile = function (svg, wallVal, row, col) {
     srcCol = (wallVal & constants.WallCoordColMask) >> constants.WallCoordColShift;
     srcWallType = (wallVal & constants.WallTypeMask) >> constants.WallTypeShift;
 
+    // Special value indicates that we should not attempt to actually draw this
+    // wall tile.
+    if (srcRow == 0xf && srcCol == 0xf) {
```
Comment (Bjvanminnen) [inline apps/src/studio/studio.js:3260]: s/==/===

**Comment by Bjvanminnen** [comment]
Can't think of any scary regressions this would cause.

File: apps/src/studio/studio.js [hunk 2]
```diff
@@ -2054,6 +2055,14 @@ var preloadActorImages = function() {
   }
 };
 
+var preloadMiscImages = function() {
+  var miscImages = skin.preloadMiscImages;
```
Comment (Bjvanminnen) [inline apps/src/studio/studio.js]: will skin.preloadMiscImages always be set?

**Pull request closed without merging**